### PR TITLE
Translate and replace @@ strings

### PR DESCRIPTION
**Pull request by maudetes**
Fix etalab/data.gouv.fr#208

Replace @@ strings by their English counterparts.
Keep English-to-French JS and Python/HTML translations in fr@@.json and gouvfr@@.po temporary files to upload on crowdin project after merge (as translations files should be created by crowdin).

I suggest we merge those @@ translations so that we can update .pot files and set up crowdin project with `main-theme` branch.

### CODE REVIEW

File: theme/js/locales/fr@@.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "You can try to reset the filters to expand your search.": "Vous pouvez essayer de réinitialiser les filtres pour agrandir votre champ de recherche.",
+  "Search for data, reuses, contributions...": "Rechercher des données, réutilisations, contributions...",
+  "Search all datasets": "Rechercher dans tous les jeux de données",
+  "See all datasets": "Voir tous les jeux de données",
+  "Search in datasets": "Rechercher dans les jeux de données",
+  "Search in reuses": "Rechercher dans toutes les réutilisations",
+  "See the reuses": "Voir toutes les réutilisations",
+  "No reuse matching your query": "Nous n’avons pas de réutilisation correspondant à votre requête",
+  "No results found.": "Aucun résultat.",
+  "You must be connected to follow.": "Vous devez être connecté pour suivre.",
```
Comment (abulte) [inline theme/js/locales/fr@@.json:67]:
```suggestion
  "You must be connected to add a favourite.": "Vous devez être connecté pour ajouter un favori.",
```

File: udata_gouvfr/theme/gouvfr/templates/dataset/resource/card.html [hunk 2]
```diff
@@ -62,19 +62,19 @@ <h4>
             {% endif %}
             <dl class="description-list">
                 <div>
-                    <dt>{{ _('@@URL') }}</dt>
+                    <dt>{{ _('URL') }}</dt>
                     <dd><a href="{{resource.url}}">{{resource.url}}</a></dd>
                 </div>
                 <div>
-                    <dt>{{ _('@@URL stable') }}</dt>
+                    <dt>{{ _('Stable URL') }}</dt>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/dataset/resource/card.html:69]:
```suggestion
                    <dt>{{ _('Permalink') }}</dt>
```

File: udata_gouvfr/theme/gouvfr/templates/dataset/resource/card.html [hunk 3]
```diff
@@ -84,15 +84,15 @@ <h4>
...
                     <dd>{{resource.created_at|dateformat(format='long')}}</dd>
                 </div>
                 <div>
-                    <dt>{{ _('@@Modifiée le') }}</dt>
+                    <dt>{{ _('Modified on') }}</dt>
                     <dd>{{resource.modified|dateformat(format='long')}}</dd>
                 </div>
                 <div>
-                    <dt>{{ _('@@Publiée le') }}</dt>
+                    <dt>{{ _('Posted on') }}</dt>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/dataset/resource/card.html:95]:
```suggestion
                    <dt>{{ _('Published on') }}</dt>
```

File: udata_gouvfr/theme/gouvfr/templates/footer.html [hunk 4]
```diff
@@ -55,8 +55,8 @@ <h4>Réseaux sociaux</h4>
                         </a>
                     </div>
                     <p class="text-black fs-sm">
-                        <span>{{ _('@@Une mission de la DINUM') }}</span>
-                        <a href="https://numerique.gouv.fr" title="{{ _('@@DINUM website') }}"
+                        <span>{{ _('A DINUM mission') }}</span>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/footer.html:58]:
```suggestion
                        <span>{{ _('A DINUM department') }}</span>
```

File: udata_gouvfr/theme/gouvfr/templates/footer.html [hunk 5]
```diff
@@ -82,7 +82,7 @@ <h4>Réseaux sociaux</h4>
         <div class="hagrid">
             <div class="row">
                 <div class="col-4 col-lg-12 marianne-logo">
-                    <a href="{{ url_for('site.home') }}" title="{{ _('@@Retourner à l\'accueil de data.gouv.fr') }}"
+                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to data.gouv.fr') }}"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/footer.html:85]:
```suggestion
                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to home page') }}"
```

File: udata_gouvfr/theme/gouvfr/templates/footer.html [hunk 6]
```diff
@@ -96,7 +96,7 @@ <h4>Réseaux sociaux</h4>
                 </ul>
                 <div class="col-2 col-lg-12 self-end text-align-right">
                     <div class="btn-secondary px-lg btn-secondary-grey-400 dropdown">
-                        <select id="lang-picker" aria-label="{{ _('@@Lang selector') }}">
+                        <select id="lang-picker" aria-label="{{ _('Lang selector') }}">
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/footer.html:99]:
```suggestion
                        <select id="lang-picker" aria-label="{{ _('Language selector') }}">
```

File: udata_gouvfr/theme/gouvfr/templates/header.html [hunk 7]
```diff
@@ -11,27 +11,27 @@
         <div class="hagrid">
             <div class="row align-items-center">
                 <div class="col-auto header-logo">
-                    <a href="{{ url_for('site.home') }}" title="{{ _('@@Retourner à l\'accueil de data.gouv.fr') }}">
+                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to data.gouv.fr') }}">
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/header.html:14]:
```suggestion
                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to home page') }}">
```

File: udata_gouvfr/theme/gouvfr/templates/header.html [hunk 8]
```diff
@@ -11,27 +11,27 @@
...
-                        title="{{ _('@@Close menu') }}" class="mobile-menu-close hidden">
-                        {{ _('@@Fermer') }}
+                        title="{{ _('Close menu') }}" class="mobile-menu-close hidden">
+                        {{ _('Close') }}
                         {% include theme('svg/close.svg') %}
                     </a>
                 </nav>
                 <div class="col-auto ml-lg ml-xl-0 col-xl-12 header-service">
-                    <a href="{{ url_for('site.home') }}" title="{{ _('@@Retourner à l\'accueil de data.gouv.fr') }}">
+                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to data.gouv.fr') }}">
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/header.html:34]:
```suggestion
                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to home page') }}">
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 9]
```diff
@@ -15,35 +15,35 @@
...
             <div class="col-5 col-lg-12">
-                <h1 class="mt-0">{{ _('@@Plateforme ouverte des données publiques Françaises') }}</h1>
+                <h1 class="mt-0">{{ _('Open platform for French public data') }}</h1>
                 <div class="mt-md mt-lg-sm">
-                    <h4>{{ _('@@Thématiques à la une') }}</h4>
+                    <h4>{{ _('Featured topics') }}</h4>
                     <a href="{{ url_for('gouvfr.spd') }}" class=" btn-venti bg-blue-470">
-                        {{ _('@@Données de') }}
-                        <strong>{{ _('@@Référence') }}</strong>
+                        {{ _('Données de') }}
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:22]:
```suggestion
                        Données de
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 10]
```diff
@@ -15,35 +15,35 @@
...
+                        <strong>{{ _('Référence') }}</strong>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:23]:
```suggestion
                        <strong>Référence</strong>
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 11]
```diff
@@ -15,35 +15,35 @@
...
                     </a>
                     <a href="{{ url_for('gouvfr.show_page', slug='donnees-coronavirus') }}"
                         class="btn-venti my-md bg-blue-450">
-                        {{ _('@@Données relatives à la') }}
-                        <strong>{{ _('@@COVID-19') }}</strong>
+                        {{ _('Données relatives à la') }}
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:27]:
```suggestion
                        Données relatives à la
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 12]
```diff
@@ -15,35 +15,35 @@
...
+                        <strong>{{ _('COVID-19') }}</strong>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:28]:
```suggestion
                        <strong>COVID-19</strong>
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 13]
```diff
@@ -15,35 +15,35 @@
...
                     </a>
                     <a href="{{ url_for('gouvfr.show_page', slug='donnees-comptes-publics') }}"
                         class="btn-venti bg-green-350">
-                        {{ _('@@Données relatives aux') }}
-                        <strong>{{ _('@@Comptes Publics') }}</strong>
+                        {{ _('Données relatives aux') }}
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:32]:
```suggestion
                        Données relatives aux
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 14]
```diff
@@ -15,35 +15,35 @@
...
+                        <strong>{{ _('Comptes Publics') }}</strong>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:33]:
```suggestion
                        <strong>Comptes Publics</strong>
```

File: udata_gouvfr/theme/gouvfr/templates/home.html [hunk 15]
```diff
@@ -122,7 +122,7 @@ <h2>{{ _('@@Données') }}</h2>
 {# Reuses section #}
 {% set nb_reuses = current_site.settings.home_reuses|length %}
 {% set reuses_tabs = (
-    ('best-reuses', _('@@Selection Etalab'), current_site.settings.home_reuses, {}),
+    ('best-reuses', _('Etalab selection'), current_site.settings.home_reuses, {}),
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/home.html:125]:
```suggestion
    ('best-reuses', _('Etalab\'s selection'), current_site.settings.home_reuses, {}),
```

File: udata_gouvfr/theme/gouvfr/templates/organization/display.html [hunk 16]
```diff
@@ -144,24 +144,24 @@ <h2>{{ _('@@Membres') }}<sup>{{ org.members|length or 0 }}</sup></h2>
         <div class="container">
             <div class="row">
                 <div class="col-6 col-md-12">
-                    <h3>{{ _('@@Informations techniques') }}</h3>
+                    <h3>{{ _('Technical Information') }}</h3>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/organization/display.html:147]:
```suggestion
                    <h3>{{ _('Technical details') }}</h3>
```

File: udata_gouvfr/theme/gouvfr/templates/organization/display.html [hunk 17]
```diff
@@ -144,24 +144,24 @@ <h2>{{ _('@@Membres') }}<sup>{{ org.members|length or 0 }}</sup></h2>
...
                             <dd class="f-bold m-0">{{ org.id }}</dd>
                         </div>
                         <div class="my-sm">
-                            <dt class="f-light">{{ _('@@Date de création') }}</dt>
+                            <dt class="f-light">{{ _('Creation date') }}</dt>
                             <dd class="f-bold m-0">{{ org.created_at|dateformat(format='long') }}</dd>
                         </div>
                         <div>
-                            <dt class="f-light">{{ _('@@Date de mise à jour') }}</dt>
+                            <dt class="f-light">{{ _('Update date') }}</dt>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/organization/display.html:158]:
```suggestion
                            <dt class="f-light">{{ _('Modification date') }}</dt>
```

File: udata_gouvfr/theme/gouvfr/templates/organization/display.html [hunk 18]
```diff
@@ -170,8 +170,8 @@ <h3>{{ _('@@Actions') }}</h3>
                         </li>
                         {% endif %}
                         <li class="my-md">
-                            <a href="{{ url_for('organizations.datasets_csv', org=org) }}" class="nav-link">{{ _('@@Télécharger
-                                la liste des jeux de données en .csv') }}</a>
+                            <a href="{{ url_for('organizations.datasets_csv', org=org) }}" class="nav-link">{{ _('Download
+                                list of .csv datasets') }}</a>
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/organization/display.html:174]:
```suggestion
                            <a href="{{ url_for('organizations.datasets_csv', org=org) }}" class="nav-link">{{ _('Download list of datasets as csv file') }}</a>
```

File: udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html [hunk 19]
```diff
@@ -19,7 +19,7 @@ <h2>
                     title="{{ _('Contact the producer') }}">{{ _('Contact the producer') }}</a></li>
             {# <li><a class="nav-link text-decoration-none text-blue-500"
                     href="{{ url_for('api.organization_followers', id=organization.id|string) }}"
-                    title="">{{ _('@@Suivre le producteur') }}</a></li> #}
+                    title="">{{ _('Follow the producer') }}</a></li> #}
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html:22]:
```suggestion
                    title="">{{ _('Add the producer as favorite') }}</a></li> #}
```

File: udata_gouvfr/theme/gouvfr/templates/post/search-result.html [hunk 20]
```diff
@@ -5,7 +5,7 @@
     <div class="result-body p-md">
         <h3 class="result-title m-0">{{ post.name }}</h3>
         <p class="fs-sm text-grey-300 my-sm">
-            {{ _('@@Publié le') }}
+            {{ _('Posted on') }}
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/post/search-result.html:8]:
```suggestion
            {{ _('Published on') }}
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 21]
```diff
@@ -0,0 +1,346 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Generated-By: Babel 2.8.0\n"
+"X-Crowdin-File: /main-theme-translate/udata_gouvfr/theme/gouvfr/translations/gouvfr@@.pot\n"
+"X-Crowdin-File-ID: 112\n"
+"X-Crowdin-Language: en\n"
+"X-Crowdin-Project: udata-gouvfr-fr-en\n"
+"X-Crowdin-Project-ID: 461514\n"
+
+#: templates/footer.html:58
+msgid "A DINUM mission"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:23]:
```suggestion
msgid "A DINUM department"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 22]
```diff
@@ -0,0 +1,346 @@
...
+msgstr "Une mission de la DINUM"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:24]:
```suggestion
msgstr "Un département de la DINUM"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 23]
```diff
@@ -0,0 +1,346 @@
...
+
+#: templates/footer.html:59
+msgid "DINUM website"
+msgstr "Le site web de la DINUM"
+
+#: templates/footer.html:85 templates/header.html:14 templates/header.html:34
+msgid "Go back to data.gouv.fr"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:31]:
```suggestion
msgid "Go back to home page"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 24]
```diff
@@ -0,0 +1,346 @@
...
+msgstr "Retourner à l'accueil de data.gouv.fr"
+
+#: templates/footer.html:99
+msgid "Lang selector"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:35]:
```suggestion
msgid "Language selector"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 25]
```diff
@@ -0,0 +1,346 @@
...
+msgstr "Sélection de langue"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:36]:
```suggestion
msgstr "Sélecteur de langue"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 26]
```diff
@@ -0,0 +1,346 @@
...
+msgid "COVID-19"
+msgstr "COVID-19"
+
+#: templates/home.html:32
+msgid "Données relatives aux"
+msgstr "Données relatives aux"
+
+#: templates/home.html:33
+msgid "Comptes Publics"
+msgstr "Comptes Publics"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:100]:
```suggestion
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 27]
```diff
@@ -0,0 +1,346 @@
...
+#: templates/home.html:87
+msgid "Reference datasets"
+msgstr "Jeux de données de référence"
+
+#: templates/home.html:95
+msgid "Data"
+msgstr "Données"
+
+#: templates/home.html:127
+msgid "Etalab selection"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:123]:
```suggestion
msgid "Etalab's selection"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 28]
```diff
@@ -0,0 +1,346 @@
...
+msgstr "Selection Etalab"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:124]:
```suggestion
msgstr "Sélection Etalab"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 29]
```diff
@@ -0,0 +1,346 @@
...
+#: templates/dataset/resource/card.html:42
+msgid "Download resource"
+msgstr "Télécharger la ressource"
+
+#: templates/dataset/resource/card.html:65
+msgid "URL"
+msgstr "URL"
+
+#: templates/dataset/resource/card.html:69
+msgid "Stable URL"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:185]:
```suggestion
msgid "Permalink"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 30]
```diff
@@ -0,0 +1,346 @@
...
+#: templates/dataset/resource/card.html:87
+msgid "Created on"
+msgstr "Créée le"
+
+#: templates/dataset/resource/card.html:91
+msgid "Modified on"
+msgstr "Modifiée le"
+
+#: templates/dataset/resource/card.html:95
+msgid "Posted on"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:205]:
```suggestion
msgid "Published on"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 31]
```diff
@@ -0,0 +1,346 @@
...
+#: templates/organization/display.html:112 templates/reuse/list.html:45
+msgid "Reuses"
+msgstr "Réutilisations"
+
+#: templates/organization/display.html:131
+msgid "Members"
+msgstr "Membres"
+
+#: templates/organization/display.html:147
+msgid "Technical Information"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:218]:
```suggestion
msgid "Technical details"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 32]
```diff
@@ -0,0 +1,346 @@
...
+msgstr "Informations techniques"
+
+#: templates/organization/display.html:154
+msgid "Creation date"
+msgstr "Date de création"
+
+#: templates/organization/display.html:158
+msgid "Update date"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:226]:
```suggestion
msgid "Modification date"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 33]
```diff
@@ -0,0 +1,346 @@
...
+
+#: templates/organization/display.html:164 templates/post/display.html:91
+#: templates/reuse/display.html:156
+msgid "Actions"
+msgstr "Actions"
+
+#: templates/organization/display.html:173
+msgid ""
+"Download\n"
+"                                list of .csv datasets"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:237]:
```suggestion
msgid "Download list of datasets as csv file"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 34]
```diff
@@ -0,0 +1,346 @@
...
+msgstr ""
+"Télécharger\n"
+"                                la liste des jeux de données en .csv"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:240]:
```suggestion
"Télécharger la liste des jeux de données en .csv"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 35]
```diff
@@ -0,0 +1,346 @@
...
+msgid "Tags"
+msgstr "Mots clés"
+
+#: templates/reuse/display.html:101
+msgid "Information"
+msgstr "Informations"
+
+#: templates/reuse/display.html:171
+msgid "Embed"
+msgstr "Intégrer"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:290]:
```suggestion
msgstr "Intégrer sur votre site"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 36]
```diff
@@ -0,0 +1,346 @@
...
+#: templates/home.html:143
+msgid "See more"
+msgstr "Voir plus"
+
+#: templates/page.html:50
+msgid "Actions"
+msgstr "Actions"
+
+#: templates/organization/producer-panel.html:22
+msgid "Follow the producer"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:325]:
```suggestion
msgid "Add the producer as favorite"
```

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po [hunk 37]
```diff
@@ -0,0 +1,346 @@
...
+msgstr "Suivre le producteur"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:326]:
```suggestion
msgstr "Ajouter le producteur en favori"
```

**Review by abulte — COMMENTED** [review]
😅 

Thanks for this hard work!

**Comment by abulte** [comment]
TODO: still some "followers" here and there, that we need to convert to favourites.

File: theme/js/components/discussions/thread.vue [hunk 38]
```diff
@@ -48,7 +48,7 @@
         v-if="_collapsed"
         @click.prevent="collapsed = false"
       >
-        {{ _discussion.length }} {{ $t("@@messages") }}
+        {{ _discussion.length }} {{ $t("messages") }}
```
[the remark this answers is not in the document]
Comment (gmougeolle) [reply theme/js/components/discussions/thread.vue:51]: ¯\_(ツ)_/¯

https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument

Looks like we must change the `$t` call to `$tc` (and change the corresponding import), then add the amount to the function call too, then handle the plural in the traduction string.

File: udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html (hunk 19 above)
Comment (maudetes) [reply udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html:22]: Favorite or Favourite, you got to pick :p

File: theme/js/components/discussions/thread.vue (hunk 38 above)
[the remark this answers is not in the document]
Comment (abulte) [reply theme/js/components/discussions/thread.vue:51]: @maudetes up to you if you think it's a good time to try that, or we can leave it for later.

File: udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html (hunk 19 above)
Comment (abulte) [reply udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html:22]: good catch 🎣

File: udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po (hunk 29 above)
Comment (abulte) [reply udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po:162]: Not sure, we're using "URL stable" a lot when talking to producers, and at least it's "real" french.

Comment (maudetes) [reply udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po]: Ok :+1:

**Review by quaxsze — APPROVED** [review]

**Pull request merged**